src/qtviewer/ParseIfcFile.cpp

```diff
@@ -0,0 +1,46 @@
+#include "ParseIfcFile.h"
+
+#include "../ifcparse/Ifc2x3.h"
+#define IfcSchema Ifc2x3
+
```

Comment:
```suggestion

```

Including a schema in the C++ code is not necessary. It only increases compile time. You're not referencing the Ifc2x3 namespace. (You shouldn't also, because it would make the code specific to a single schema.)